cellprofiler/modules/measureimageskeleton.py

```diff
@@ -0,0 +1,156 @@
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import scipy.ndimage
+import skimage.measure
+import skimage.segmentation
+import skimage.util
```

Comment:
See the second bullet point under the [imports section of PEP08](https://www.python.org/dev/peps/pep-0008/#imports). PyCharm can do this for you if you navigate to Code > Optimize Imports.